src/sinks/clickhouse.rs

```diff
@@ -25,7 +25,7 @@ pub struct ClickhouseConfig {
     pub host: String,
     pub table: String,
     pub database: Option<String>,
-    pub compression: Option<Compression>,
+    pub compression: Compression,
```

Comment:
I think this needs a specific serde default here so that it doesn't accidentally become required (breaking existing configs) and defaults to gzip (also kinda breaking).

Reply:
Oh, skipped this. Why defaults to gzip? Currently default is `none` and `#[serde(default)]` should be enough, because `Compression::None` is default.

Reply:
The current default is `None`, because the field is an `Option<Compression>`. When it is unwrapped in `fn build`, the gzip default is applied. I do note that the existing documentation does not reflect this default. @lukesteensen you are the original author of this sink, maybe you could comment why it defaults to gzip compression.

Reply:
Ohhh, I missed this! Messed up between changes. I'll change it then, fix docs and it will be possible switch to `none` if required in another PR.

Reply:
Fixed.